Johnpaul-Borree/Fast-Food-Fast
# #160495513 Ch-mocha-test1

#### What does this PR do?
Test Api endpoint for create order
#### Description of Task to be completed?
- Create file to test Endpoint: Post /ap1/v1/orders
- Create tests to test for creating order taking sample order
- create tests for validating this endpoints
#### How should this be manually tested?
- Run command npm run tests 
- check continuous integrations 
- look out for readme badge on this repo
#### What are the relevant pivotal tracker stories?
#160495513
